intel_extension_for_transformers/neural_chat/pipeline/plugins/retrieval/retrieval_agent.py

```diff
@@ -134,9 +134,11 @@ def __init__(self,
 
         if self.vector_database == "Chroma":
             self.database = Chroma()
+        elif self.vector_database == "Qdrant":
+            self.database = Qdrant
```

Comment:
add a "()", should keep the same format as Chroma

Reply:
Qdrant init required 2 positional arguments: 'client' and 'collection_name'. Only @classmethod functions will be called below, so maybe we can avoid initializing the vectorstore classes here?